app/services/organizations/update_service.rb

```diff
@@ -120,6 +120,12 @@ def handle_legacy_vat_rate(vat_rate)
     end
 
     def handle_eu_tax_management(eu_tax_management)
+      org_outside_eu = !organization.eu?
+      trying_to_enable_eu_tax_management = params[:eu_tax_management] && !organization.eu_tax_management
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. (https://rubystyle.guide#if-as-a-modifier)